### ISSUE TITLE
gopls: automated issue report (crash when updating 'go')

### ISSUE THREAD

**Issue by FranckEtienne** [golang/vscode-go#1949]
gopls version: v0.7.4
gopls flags: 
update flags: proxy
extension version: 0.27.2
go version: 1.17.3
environment: Visual Studio Code win32
initialization error: undefined
manual restart count: 0
total start count: 2

ATTENTION: PLEASE PROVIDE THE DETAILS REQUESTED BELOW.

Describe what you observed.

<ANSWER HERE>

Please attach the stack trace from the crash.
A window with the error message should have popped up in the lower half of your screen.
Please copy the stack trace and error messages from that window and paste it in this issue.

<PASTE STACK TRACE HERE>

Failed to auto-collect gopls trace: unrecognized crash pattern.

OPTIONAL: If you would like to share more information, you can attach your complete gopls logs.

NOTE: THESE MAY CONTAIN SENSITIVE INFORMATION ABOUT YOUR CODEBASE.
DO NOT SHARE LOGS IF YOU ARE WORKING IN A PRIVATE REPOSITORY.

<OPTIONAL: ATTACH LOGS HERE>

**Comment by hyangah** [comment]
Thanks for the report. Can you please share the log you see in the gopls output channel and tell us when you observed this popup (e.g. I was updating `gopls`, I just opened the vscode, I did refactoring, ...)?
![Screen Shot 2021-01-27 at 2 53 49 PM](https://user-images.githubusercontent.com/4999471/106151310-3899a680-614a-11eb-9be7-42f2ae0592c9.png)

**Comment by FranckEtienne** [comment]
Hi all,

Sorry, I relaunched vscode so I haven’t keep the traces expected.

What I can say is that I update Go from 1.17.3 to 1.17.5 without relaunch vscode.
It is perhaps the origine of error.

Cordialement/Best regards
***@***.***

Franck ETIENNE
Téléphone : +33 (0) 4 38 12 54 93
***@***.******@***.***>



De : Hyang-Ah Hana Kim ***@***.***>
Envoyé : jeudi 16 décembre 2021 17:34
À : golang/vscode-go ***@***.***>
Cc : ETIENNE, FRANCK ***@***.***>; Author ***@***.***>
Objet : Re: [golang/vscode-go] gopls: automated issue report (crash) (Issue #1949)


Caution! External email. Do not open attachments or click links, unless this email comes from a known sender and you know the content is safe.

Thanks for the report. Can you please share the log you see in the gopls output channel and tell us when you observed this popup (e.g. I was updating gopls, I just opened the vscode, I did refactoring, ...)?
[Screen Shot 2021-01-27 at 2 53 49 PM]<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F4999471%2F106151310-3899a680-614a-11eb-9be7-42f2ae0592c9.png&data=04%7C01%7Cfranck.etienne%40atos.net%7Ccc6bde757b5f4cd990a808d9c0b1dbcc%7C33440fc6b7c7412cbb730e70b0198d5a%7C0%7C0%7C637752692409000095%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=G1GDv4cB8sbBhXmQVZRLBVtpu%2BrhCVEzGcJdY%2BmUxnE%3D&reserved=0>

—
Reply to this email directly, view it on GitHub<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fgolang%2Fvscode-go%2Fissues%2F1949%23issuecomment-995981618&data=04%7C01%7Cfranck.etienne%40atos.net%7Ccc6bde757b5f4cd990a808d9c0b1dbcc%7C33440fc6b7c7412cbb730e70b0198d5a%7C0%7C0%7C637752692409010055%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=7AjZ%2FqdwklV%2BUbsP2sc%2BDK6rGs03qBzBeQVJ1nqNDqg%3D&reserved=0>, or unsubscribe<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACDWMQNA7A2C4JLV3RVKG2TURIIGBANCNFSM5KGW6SXQ&data=04%7C01%7Cfranck.etienne%40atos.net%7Ccc6bde757b5f4cd990a808d9c0b1dbcc%7C33440fc6b7c7412cbb730e70b0198d5a%7C0%7C0%7C637752692409010055%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=R%2BcvZB5i7tAvAX7BguS4%2F%2BExJeJakLw4Yil5N5ocjzE%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cfranck.etienne%40atos.net%7Ccc6bde757b5f4cd990a808d9c0b1dbcc%7C33440fc6b7c7412cbb730e70b0198d5a%7C0%7C0%7C637752692409020004%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=XJDDxrdnlo%2BIb3pd64YoM%2Be7dN2bg1JjvFGsvIy3iII%3D&reserved=0> or Android<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cfranck.etienne%40atos.net%7Ccc6bde757b5f4cd990a808d9c0b1dbcc%7C33440fc6b7c7412cbb730e70b0198d5a%7C0%7C0%7C637752692409020004%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=p0d0jzp1O6KQTN9wyK2beGJIpEH2OP6iPJ9aDHHU5mE%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

Ce message et toutes les pièces jointes (ci-après le "message") sont établis à l’intention exclusive des destinataires désignés. Il contient des informations confidentielles et pouvant être protégé par le secret professionnel. Si vous recevez ce message par erreur, merci d'en avertir immédiatement l'expéditeur et de détruire le message. Toute utilisation de ce message non conforme à sa destination, toute diffusion ou toute publication, totale ou partielle, est interdite, sauf autorisation expresse de l’émetteur. L'internet ne garantissant pas l'intégrité de ce message lors de son acheminement, Atos (et ses filiales) décline(nt) toute responsabilité au titre de son contenu. Bien que ce message ait fait l’objet d’un traitement anti-virus lors de son envoi, l’émetteur ne peut garantir l’absence totale de logiciels malveillants dans son contenu et ne pourrait être tenu pour responsable des dommages engendrés par la transmission de l’un d’eux.

This message and any attachments (the "message") are intended solely for the addressee(s). It contains confidential information, that may be privileged. If you receive this message in error, please notify the sender immediately and delete the message. Any use of the message in violation of its purpose, any dissemination or disclosure, either wholly or partially is strictly prohibited, unless it has been explicitly authorized by the sender. As its integrity cannot be secured on the internet, Atos and its subsidiaries decline any liability for the content of this message. Although the sender endeavors to maintain a computer virus-free network, the sender does not warrant that this transmission is virus-free and will not be liable for any damages resulting from any virus transmitted.

**Comment by hyangah** [comment]
Thanks! There is an issue related to handling go binary change during a session more gracefully (https://github.com/golang/go/issues/49883), but we need more info to see if this is the same issue. Closing for now.

Please open a new issue when you encounter a crash.
